CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+## Version 0.37.0
+
+<!-- ([Full Changelog](https://github.com/unitaryfund/mitiq/compare/v0.36.0...v0.37.0)) -->
+
+### Highlights
+
+✨ Stacking quantum error mitigation techniques is a primary area of focus in Mitiq. In this release, @jordandsullivan introduced a **Tutorial on composing Digital Dynamical Decoupling (DDD) and Zero Noise Extrapolation (ZNE)**.
+
+🗒️ **»Download Notebook«**: Users have now the option to download tutorials in Jupyter `.ipynb` format. We hope this will encourage experimentation with existing tutorials.
```

Comment:
```suggestion
🗒️ **»Download Notebook«**: Users have now the option to download tutorials in Jupyter `.ipynb` format directly from our documentation. We hope this will encourage experimentation with existing tutorials.
```

I wanted to clarify where you can download from, but the phrasing is still a little awkward. Do you think it's needed? Maybe it's obvious.

Reply:
It's an improvement, I committed the suggestion. Thanks.